spdx/relationship.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2020 Yash Varshney
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from enum import Enum
+
+# Implement the auto feature that becomes available in 3.6
+autoinc = 0
+
+
+def auto():
+    global autoinc
+    autoval = autoinc
+    autoinc += 1
+    return autoval
+
```

Comment:
We could also use enum.auto() function from enum module, instead of creating our own.

Reply:
The `auto` feature is supported only after 3.6 version of python and presently python-tools does not support it. i.e. why I made the same auto feature here.